.github/CODEOWNERS

```
@@ -145,6 +145,8 @@
 /pkg/autodiscovery/listeners/           @DataDog/container-integrations
 /pkg/autodiscovery/listeners/cloudfoundry*.go  @DataDog/integrations-tools-and-libraries
 /pkg/autodiscovery/listeners/snmp*.go   @DataDog/infrastructure-integrations
+/pkg/autodiscovery/providers/           @DataDog/container-integrations
+/pkg/autodiscovery/providers/file*.go   @DataDog/container-integrations @DataDog/agent-core
```

I think only agent-core is needed for this one

```suggestion
/pkg/autodiscovery/providers/file*.go   @DataDog/agent-core
```